### ISSUE TITLE
Audio Output Device Select Option missing

### ISSUE THREAD

**Issue by ajayalag1974** [mynaparrot/plugNmeet-client#461]
Hi,

Just like there exist an option to select microphone & camera from available list of microphones & cameras, option to select Audio Output Device from available list of Audio Output Devices like speakers, headphones, virtual output, etc etc.

Jitsi have this option so yeah its possible.

**Comment by jibon57** [comment]
Yes, contributors are welcome to implement. You can also contribute by sending pull request.

**Comment by jibon57** [comment]
Probably you'll need to follow: https://www.plugnmeet.org/docs/developer-guide/setup-development#troubleshooting
